cmd/utils/flags.go

```diff
@@ -214,7 +214,7 @@ var (
 	}
 	DynamoDBTableNameFlag = cli.StringFlag{
 		Name:  "db.dynamo.tablename",
```

Comment:
Don't we need to change `db.dynamo` to `db.dynamos3`?

Reply:
If a user wants to use dynamoDB, one needs to set `--dbtype DynamoDBS3`. I thought this change will alert users.
Do you think changing `db.dynamo` to `db.dynamos3` is necessary in all codes?

Reply:
Though this is not that necessarily, I think this can confuse users because if you want to use dynamodb, you need to user the word "dynamos3". However, to set up configuration, what used is "dynamo". I think it's up to you 😅